tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -150,62 +159,99 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     };
 
     fluid.tests.tooltipModuleSource = function (pager) {
-        var pageLinksTop = $("a", pager.pagerBar.locate("pageLinks"));
-        var pageLinksBottom = $("a", pager["pagerBar-1"].locate("pageLinks"));
-
-        var tooltipContents = [
-            [
-                {nodeName: "b", nodeText: "bird"},
-                {nodeName: "b", nodeText: "fish"}
-            ]
-        ];
-
         var sequence = [];
+        var linksList = pager.dataModel.map(function(data) {
+            return {text: data.animal};
+        });
 
-        var assertVisibleTips = function (message, targetIds) {
+        /* Change the page size and verify whether the tooltips and links are synced accordingly */
+        for (var pageSize = 2; pageSize <= 4; pageSize++) {
             sequence.push({
-                event: "{trackTooltips}.events.notifyFocusChange",
-                listener: function () {
-                    fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
-                        jqUnit.assertNode("The contents of the tooltip should be set", tooltipContents[0], $("b", tooltip));
-                    });
-                }
+                func: "fluid.tests.setPageSize",
+                args: [pager, pageSize]
```

Comment:
You can avoid using `fluid.tests.setPageSize` here with the following IoC expression.

```javascript
{
    func: "fluid.changeElementValue",
    args: ["pager.dom.pageSize", pageSize]
}
```